profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -0,0 +1,413 @@
+package com.telecominfraproject.wlan.profile.rf.models;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.telecominfraproject.wlan.core.model.equipment.AutoOrManualValue;
+import com.telecominfraproject.wlan.core.model.equipment.ChannelBandwidth;
+import com.telecominfraproject.wlan.core.model.equipment.ChannelHopSettings;
+import com.telecominfraproject.wlan.core.model.equipment.RadioBestApSettings;
+import com.telecominfraproject.wlan.core.model.equipment.RadioType;
+import com.telecominfraproject.wlan.core.model.json.BaseJsonModel;
+import com.telecominfraproject.wlan.equipment.models.ActiveScanSettings;
+import com.telecominfraproject.wlan.equipment.models.ManagementRate;
+import com.telecominfraproject.wlan.equipment.models.MimoMode;
+import com.telecominfraproject.wlan.equipment.models.MulticastRate;
+import com.telecominfraproject.wlan.equipment.models.NeighbouringAPListConfiguration;
+import com.telecominfraproject.wlan.equipment.models.StateSetting;
+
+public class RfElementConfiguration extends BaseJsonModel {
+
+	private static final long serialVersionUID = 1071268246965238451L;
+	
+    private static final Logger LOG = LoggerFactory.getLogger(RfElementConfiguration.class);
+    
+    public final static int MIN_BG_RADIO_CELL_SIZE = -80;
+	public final static int MIN_AC_RADIO_CELL_SIZE = -80;
+	public final static int DEFAULT_RX_CELL_SIZE_DB = -90;
+	public final static int DEFAULT_EIRP_TX_POWER = 18;
+	public final static int DEFAULT_BEACON_INTERVAL = 100;
+	
+    // Parameters for each RadioType
+    private String rf;
+	private Integer beaconInterval; // keep this in sync with fields in RadioCfg (in protobuf)
+    private StateSetting forceScanDuringVoice;
+    private Integer rtsCtsThreshold;
+    private ChannelBandwidth channelBandwidth;
+    private MimoMode mimoMode;
+    private Integer maxNumClients;
+    private MulticastRate multicastRate;
+    private boolean autoChannelSelection;
```

Comment:
Do we still want a separate flag for autoChannelSelection? Or just a single flag for autoRrm (to choose between auto or manual RRM)